.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+# Description
+
+
+# Issue
```

Comment:
```suggestion
## Issue
```

It is better to use h2 or subheadings for headers that not the first one

Reply:
All these items are same level of headings.

Reply:
Yes, but `#` is equal to `<h1>` and `##` is equal to `<h2>`, but for markdown and HTML, it is recommended to have only 1 `<h1>` as the title. It seems that although it was originally done with h2 in the PR, they were changed to h1 without me even realizing it.

Reply:
should `# Description` also be `## Description`?

Reply:
No, since it would be the "title" of the PR, so it would have `<h1>`

Reply:
it's a bit confusing since markdown is sort of a superset of HTML that is supposed to be able to be rendered to HTML but has some weird stuff.

It would be fine to ignore this since we used `<h1>` for all of the fields in SVT-AV1